opentelemetry-sdk/CHANGELOG.md

```diff
@@ -2,16 +2,20 @@
 
 ## Unreleased
 
-- Rename `MetricRecord` class to `ExportRecord`
-  ([#1367](https://github.com/open-telemetry/opentelemetry-python/pull/1367))
+- Allow samplers to modify tracestate
+  ([#1319](https://github.com/open-telemetry/opentelemetry-python/pull/1319))
 - Add optional parameter to `record_exception` method
   ([#1314](https://github.com/open-telemetry/opentelemetry-python/pull/1314))
 - Update exception handling optional parameters, add escaped attribute to record_exception
   ([#1365](https://github.com/open-telemetry/opentelemetry-python/pull/1365))
+- Rename `MetricRecord` class to `ExportRecord`
+  ([#1367](https://github.com/open-telemetry/opentelemetry-python/pull/1367))
 - Rename Record in Metrics SDK to Accumulation
   ([#1373](https://github.com/open-telemetry/opentelemetry-python/pull/1373))
 - Rename Meter class to Accumulator in Metrics SDK
   ([#1372](https://github.com/open-telemetry/opentelemetry-python/pull/1372))
+- Rename Meter class to Accumulator in Metrics SDK
+  ([#1372](https://github.com/open-telemetry/opentelemetry-python/pull/1372))
```

Comment:
Double entry here! Otherwise this looks great :)

Reply:
Didn't realize it was merged, opened this PR: https://github.com/open-telemetry/opentelemetry-python/pull/1414